src/analysis/retail/evoker/preservation/modules/talents/EmeraldCommunion.tsx

```diff
@@ -150,10 +150,9 @@ class EmeraldCommunion extends Analyzer {
             let targetsHitPerf = QualitativePerformance.Good;
             // only check lifebind targets if they are echo build
             if (this.selectedCombatant.hasTalent(TALENTS_EVOKER.STASIS_TALENT)) {
-              const percentHit = info.numLifebinds / info.possibleTargets;
-              if (percentHit < 0.7) {
+              if (info.numLifebinds < Math.min(11, info.possibleTargets)) {
                 targetsHitPerf = QualitativePerformance.Fail;
-              } else if (percentHit < 0.8) {
+              } else if (info.numLifebinds < Math.min(15, info.possibleTargets)) {
```

Comment:
Put these arbitrary thresholds into constants at the top of the file or in the shared constants file